app/views/users/sessions/new.html.erb

```diff
@@ -1,29 +1,61 @@
-<h2>Log asd in</h2>
-
+<main class="form-signin text-center">
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  
-  <div class="field">
+  <%= image_tag("login_logo.jpg", :alt => "rss feed",:width => "72", :height => "57") %>
+    <h1 class="h3 mb-3 fw-normal">Please sign in</h1>
+
+  <div class="form-floating">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email",class:"form-control",placeholder:"Email" %>
     <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
   </div>
 
- 
-
-  <div class="field">
+  <div class="form-floating">
+    <%= f.password_field :password, autocomplete: "current-password",class:"form-control",placeholder:"Password",focus: true %>
     <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
-  </div>
+    </div>
 
   <% if devise_mapping.rememberable? %>
-    <div class="field">
+    <div class="checkbox mb-3">
       <%= f.check_box :remember_me %>
       <%= f.label :remember_me %>
     </div>
   <% end %>
 
   <div class="actions">
-    <%= f.submit "Log in" %>
+    <%= f.submit "Sign in",class:"w-100 btn btn-lg btn-primary" %>
   </div>
 <% end %>
+</main>
 
 <%= render "users/shared/links" %>
+
+
+<style type="text/css">
```

Comment:
@rubyboy-hub  Hi, The css code needs to be moved to corresponding stylesheet folder inside app/assets.  Normally, we don't write the css in erb files.

Reply:
Okay , I will add the association and  change the respective css file  into a separate css file